test/CloudNative.CloudEvents.UnitTests/Avro/Helpers/FakeGenericRecordSerializer.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Avro;
```

Comment:
Please add a copyright header.

Reply:
Added. I've also added it to `BasicGenericRecordSerializer.cs`